scholia/app/templates/q_curation.html

```diff
@@ -0,0 +1,32 @@
+{% extends "base.html" %}
+
+{% block page_content %}
+
+<h1 id="h1">Improve...</h1>
+
+<p>Our data source is <a href="https://www.wikidata.org/wiki/Wikidata:Main_Page">Wikidata</a>, a structured data project
+  similar to the encyclopedia project Wikipedia. Each item is identified by a Q number, and information can be added
+  directly to an item's page (in this case, <a
+    href='https://www.wikidata.com/{{q}}'>https://www.wikidata.com/{{q}}</a>). Some information however, like an
+  author's papers, are stored on the item for the paper itself. The "<a
+    href="https://www.wikidata.org/w/index.php?title=Special%3AWhatLinksHere&target{{q}}&namespace=0">What links
+    here</a>" link in the sidebar of an item can show these items.</p>
+
+{% if self.wikidata() %}
+{% endif %}
+
+{% block wikidata %}{% endblock %}
+
+<p>There are also various <a href="https://www.wikidata.org/wiki/Wikidata:Tools/Edit_items">tools</a> available which
+  can make editing Wikidata quicker and more rigorous.</p>
+
+{% if self.author_disambiguator() %}
+<p>Most importantly for Scholia is the <a href="https://author-disambiguator.toolforge.org">Author
+    Disambiguator</a>, which can be used to quickly create new authors and find
```

Comment:
Wikidata is probably not in the business of creating authors. Perhaps ```create dedicated items for authors```.

Reply:
Isn't this the fastest way to create an author? 
![image](https://user-images.githubusercontent.com/6676843/128790170-681e67cc-2dd2-466f-9a5b-4b98a405fe12.png)